docs/source/storeconv.md

```diff
@@ -115,56 +119,62 @@ be represented fully by splitting datasets in this way. In these cases:
 opener does not support varying values of one of these parameters, a constant 
 value must defined. This may be achieved by the JSON schema's `const` property 
-or by an `enum` property whose value is a one-element array.
+or by an `enum` property value which is a one-element array.
 
 Any dataset requiring the specification of these parameters MUST
 use the standard parameter names, syntax, and semantics defined below, in
 order to keep the interface consistent. For instance, if a dataset allows a
 time aggregation period to be specified, it MUST use the `time_period`
 parameter with the format described below rather than some other alternative
-name and/or format. The parameters are described below with their Python type
+name and/or format. Below, the parameters are described with their Python type
 annotations.
 
  - `variable_names: List[str]`  
    A list of the identifiers of the requested variables. 
    This parameter MUST be included in an opener parameters schema.
+   
  - `bbox: Union[str,Tuple[float, float, float, float]]`
    The bounding box for the requested data, in the order xmin, ymin, xmax, ymax. 
    Must be given in the units of the specified spatial coordinate reference system `crs`.
    This parameter MUST be included in an opener parameters schema. 
+   
  - `crs: str`  
    The identifier for the spatial coordinate reference system of geographic data.
+   
  - `spatial_res: float`  
    The requested spatial resolution (x and y) of the returned data.
    Must be given in the units of the specified spatial coordinate reference system `crs`.
    This parameter MUST be included in an opener parameters schema. 
+   
  - `time_range: Tuple[Optional[str], Optional[str]]`  
    The requested time range for the data to be returned. See section
    ‘[Date, time, and duration specifications](#sec-datespec)’.
    This parameter MUST be included in an opener parameters schema.
+   
  - `time_period: str`  
    The requested temporal aggregation period for the data. See section
    ‘[Date, time, and duration specifications](#sec-datespec)’.
    This parameter MUST be included in an opener parameters schema.
+   
  - `force_cube: bool`  
    Whether to return results as a [specification-compliant
    xcube](https://github.com/dcs4cop/xcube/blob/master/docs/source/cubespec.md).
    If a store supports this parameter and if a dataset is opened with this
    parameter set to `True`, the store MUST return a specification-compliant
-   xcube. If this parameter is not supported or if a dataset is opened with
+   xcube dataset. If this parameter is not supported or if a dataset is opened with
    this parameter set to `False`, the caller MUST NOT assume that the returned
    data conform to the xcube specification.
 
 ### Semantics of list-valued parameters
 
-The `variables` parameter takes as its value a list, with no members repeated
+The `variables` parameter takes as its value a list, with no members duplication
```

Comment:
Ungrammatical. "member duplication" (without "s") would be grammatically correct, but I would find "duplicated members" clearer.

Reply:
Or just "duplicate members"